gui/GiveItem.java

```diff
@@ -82,21 +82,11 @@ private void giveItemToPlayer() {
 		}
 		String itemMod = "";
 		if (itmSt != null) {
-			itemMod = "" + itmSt.getItemDamage();
+			itemMod = " " + itmSt.getItemDamage();
 		}
-		String amt;
-		if (!((amountBox.getText() == null) || amountBox.getText().isEmpty())) {
-			try {
-				int amount = Math.abs(Integer.parseInt(amountBox.getText())) % 65;
-				amt = "" + amount;
-			} catch (NumberFormatException nfe) {
-				amt = "1";
-			}
-		} else {
-			amt = "1";
-		}
-		PacketDispatcher.sendToServer(new MentorCommandMessage(
-				"/give " + userBox.getText() + " " + tItem.getRegistryName() + " " + amt + " " + itemMod));
+		String amt = amountBox.getText() == null ? "1" : amountBox.getText();
```

Comment:
You have to use the mentor command message to do this as mentors no longer have certain privilages, see the changes <<